packages/react/src/components/dropdown/combobox/Combobox.tsx

```diff
@@ -500,6 +505,16 @@ export const Combobox = <OptionType,>(props: ComboboxProps<OptionType>) => {
         >
           <IconAngleDown className={styles.angleIcon} aria-hidden />
         </button>
+        {showClearButtonForSingleSelect && (
+          <button
+            type="button"
+            className={classNames(selectStyles.clearButton, !showToggleButton && selectStyles.noToggle)}
+            onClick={() => resetCombobox()}
```

Comment:
I'm wondering where the focus should move after the clear button is pressed. Now the focus disappears. Maybe need to ask this from the accessibility team?

Reply:
In multiselect the focus disappeared, so I went with the existing behaviour.

Reply:
- [ ] Focus Combobox/Select after clear button press